org-formation/_parameters.yaml

```diff
@@ -16,3 +16,7 @@ allRegions:
   Type: List<String>
   Default:
     - us-east-1
+
+budgetOwnerEmail:
+  Type: String
+  Default: bruce.hoff@sagebase.org
```

Comment:
Seems like this should be an email alias (a Google group in the sagebase.org domain) that individuals could subscribe to / unsubscribe from.

Reply:
OK.  who should be in the alias @brucehoff ?

Reply:
I need more context to answer your question, @zaro0508 .  Who asked for this?  Who is the one responsible for controlling costs incurred in the "mobile health data engeering account"?  It seems like that person would want to subscribe to the alias.

Reply:
set to send @tthyer budget notifications